resources/Materials/TestSuite/stdlib/texture/image_addressing.mtlx

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="uclamp" type="" xpos="8.43448" ypos="4.84">
+    <image name="image_number_1" type="color3" xpos="5.36637" ypos="4.71876">
```

Comment:
This tests the bad parsing logic I had for image name. Uniform created will be called image_number_1_file so we take away the _file and append strings list uaddressmode. Since I parsed the string incorrectly I would be looking for image_uaddressmode which would not exist.